cfme/infrastructure/pxe.py

```diff
@@ -463,15 +463,23 @@ class SystemImageType(Updateable, Pretty):
     """
     appliance = CurrentAppliance()
     pretty_attrs = ['name', 'provision_type']
+    VM_OR_INSTANCE = "VM and Instance"
+    HOST_OR_NODE = "Host / Node"
 
     def __init__(self, name=None, provision_type=None, appliance=None):
         self.appliance = appliance or get_or_create_current_appliance()
         self.name = name
         self.provision_type = provision_type
 
     def _form_mapping(self, create=None, **kwargs):
+        if kwargs.get('provision_type') == self.VM_OR_INSTANCE:
+            provision_type = "VM and Instance"
```

Comment:
Yes, whatever gets passed shall get to the UI, we can't make this the API to the product, protecting from the errors :smile: